gmmp/settings.py

```diff
@@ -204,10 +204,11 @@
     "QN": {'name': _('Belgium - Flemish'), 'alpha3': 'VLG'},
     "UK": None,
     "QO": {'name': _('England'), 'alpha3': 'ENG'},
-    "QP": {'name': _("North Ireland"), 'alpha3': 'NIR'},
+    "QP": {'name': _("Northern Ireland"), 'alpha3': 'NIR'},
     "QQ": {'name': _("Scotland"), 'alpha3': 'SCT'},
     "QR": {'name': _("Wales"), 'alpha3': 'WLS'},
     "XI": {'name': _("International"), 'alpha3': 'XIN'},
+    "PS": {'name': _("Palestine"), 'alpha3': 'PSE'}
```

Comment:
Does region mapping for `Palestine` already exist?

Reply:
yes. under the name `Palestine, State of`

Reply:
Do they use the same `code` @esirK ?

Reply:
@kilemensi 
my understanding is the change is to prevent showing the user `Palestine, State of` on the UI, right? If so, that's why we don't have a new region called `Palestine`. 
Also from `django_countries`, we don't have such a region https://github.com/SmileyChris/django-countries/blob/b4869ad4e6/django_countries/data.py#L202

Reply:
Yes @esirK & hence why I asked about the `code`... i.e. country code and not region code. Everything uses `code` so as long as `code` is the same, the name can be anything.

Reply:
In that case yes `PS` is the `country code` for both.

Reply:
@kilemensi  I think we can merge it now yes?